geo-types/src/macros.rs

```diff
@@ -55,8 +55,22 @@ macro_rules! point {
 /// [`Coordinate`]: ./struct.Point.html
 #[macro_export]
 macro_rules! coord {
-    (x: $x:expr, y: $y:expr $(,)?) => {
-        $crate::Coordinate { x: $x, y: $y }
+    (x: $x:expr, y: $y:expr $(,)* ) => {
+        $crate::Coordinate::new__(
+            $x,
+            $y,
+            $crate::NoValue::default(),
```

Comment:
```suggestion
            $crate::NoValue,
```
(and similar elsewhere)

Reply:
thx, didn't know empty structs can be instantiated without the default. Fixed.